kafka-java-console-sample/docs/Local.md

```diff
@@ -94,15 +94,14 @@ We will not discuss establishing a connection from your laptop to Event Streams
 
    The `java -jar ./build/libs/kafka-java-console-sample-2.0.jar` part of the command identifies the locations of the .JAR file to run within the cloned repository. You do not need to change this. 
    
-   Use the `kafka_brokers_sasl` from the **Service credentials** created in Step 2. We recommend using all the `kafka_brokers_sasl` listed in the **Service credentials** that you created.
+   Use the `kafka_brokers_sasl` from the **Service credentials** created in Step 2. We recommend using only one of the `kafka_brokers_sasl` at first and then growing to multiple brokers later, this makes running the samples much simpler however it is important to use all the `kafka_brokers_sasl` in your production workloads
```

Comment:
Hi @Gorgonx7, what's your thinking behind this recommendation? Is it that the `kafka_brokers_sasl` property is annoying to combine into the right format? If so, I think the recommendation should be to use the (new-ish) `bootstrap_endpoints` property - which is already formatted in an appropriate way.

I'm reluctant to suggest "only use a single bootstrap endpoint", even for initial testing, as I think it is something that's easily forgotten about (at least until it becomes a problem).